cli/script/command-executor.ts

```diff
@@ -676,6 +826,29 @@ function getPackageString(packageObject: Package): string {
         (packageObject.description ? wordwrap(70)("\n" + chalk.green("Description: ") + packageObject.description): "");
 }
 
+function getPackageMetricsString(packageObject: PackageWithMetrics): string {
+    if (!packageObject || !packageObject.metrics) {
+        return "";
+    }
+    
+    var percentString: string = packageObject.metrics.activePercent === 100.0 ? "100" : packageObject.metrics.activePercent.toPrecision(2) + "%";
+    var numPending: number = packageObject.metrics.downloaded - packageObject.metrics.installed - packageObject.metrics.failed;
+    var returnString: string = chalk.green("Active: ") + percentString + " (" + packageObject.metrics.active + ")";
+    if (packageObject.metrics.installed || numPending) {
```

Comment:
I'm in favor of showing the installs metric even if it's zero, but only showing the pending number of its non-zero.